### PR TITLE
Support global-workflow GEFS C48 on Google Cloud

### LINKED ISSUE #2860: Support gefs C48 on Google
### What new functionality do you need?

Support global-workflow GEFS C48 on Google Cloud.

### What are the requirements for the new functionality?

Make changes to Google cloud env and yaml files so gefs can run.

### Acceptance Criteria

All on-prem tests and runs still work properly, and gefs case run succeed on Google  Cloud.

### Suggest a solution (optional)

Have already checked on EPIC branch, need file PR and test to merge back to EMC repo.

### PR DESCRIPTION
**Pull request by weihuang-jedi**
Support global-worflow GEFS C48 on Google Cloud.

# Description

Make env. var. and yaml file changes, so global-workflow GEFS C48 case can run properly on Google Cloud.

  Resolves #2860

# Type of change

- New feature (adds functionality)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? YES
- Does this change require an update to any of the following submodules? NO (If YES, please add a link to any PRs that are pending.)
  - [ ] EMC verif-global
  - [ ] GDAS
  - [ ] GFS-utils
  - [ ] GSI
  - [ ] GSI-monitor
  - [ ] GSI-utils
  - [ ] UFS-utils
  - [ ] UFS-weather-model
  - [ ] wxflow

# How has this been tested?

- Clone and build on Google Cloud
- GEFS C48 on Google cloud

# Checklist
- [x ] Any dependent changes have been merged and published
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings
- [ x] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Review by aerorahul — DISMISSED** [review]
looks good.

File: parm/config/gefs/config.base [hunk 1]
```diff
@@ -345,9 +345,12 @@ export DELETE_COM_IN_ARCHIVE_JOB="YES"   # NO=retain ROTDIR.  YES default in arc
...
 
-# The tracker, genesis, and METplus jobs are not supported on AWS yet
-# TODO: we should place these in workflow/hosts/awspw.yaml as part of AWS setup, not for general.
-if [[ "${machine}" == "AWSPW" ]]; then
+# The tracker, genesis, and METplus jobs are not supported on CSPs yet
+# TODO: we should place these in workflow/hosts/[csp]pw.yaml as part of AWS/AZURE/GOOGLE setup, not for general.
+if [[ "${machine}" =~ "PW" ]]; then
+   export DO_TRACKER="NO"
+   export DO_GENESIS="NO"
+   export DO_METP="NO"
```
Comment (WalterKolczynski-NOAA) [inline parm/config/gefs/config.base]: GEFS already doesn't run any of these.
```suggestion
```

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

File: env/GOOGLEPW.env [hunk 2]
```diff
@@ -43,6 +43,17 @@ if [[ "${step}" = "fcst" ]] || [[ "${step}" = "efcs" ]]; then
     export APRUN_UFS="${launcher} -n ${ufs_ntasks}"
     unset nnodes ufs_ntasks
 
+elif [[ "${step}" = "prep_emissions" ]]; then
+
+    export APRUN="${APRUN}"
```
Comment (aerorahul) [inline env/GOOGLEPW.env]:
```suggestion
    export APRUN
```
should be sufficient, i.e. there is no need to assign the variable to itself to export it.

**Review by aerorahul — DISMISSED** [review]
looks good.

File: env/GOOGLEPW.env (hunk 2 above)
Comment (weihuang-jedi) [reply env/GOOGLEPW.env]: Agreed, and changed.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**